vendor/github.com/giantswarm/e2e-harness/pkg/framework/util.go

```diff
@@ -14,11 +12,11 @@ import (
 )
 
 const (
-	defaultTimeout = 600
+	defaultTimeout = 500
```

Comment:
I changed this when the integration tests took longer and failed with a timeout. We have to keep an eye on this and see if this is cool again.